### DISCUSSION TITLE
language filtering

### DISCUSSION THREAD

**Discussion by fluffier76** [ReVanced/revanced-discussions#1075] [Question] [answered]
Hello, I want my kid to learn english, I would like his youtube to only show him english content, so any videos whose language is not english I would like them not to be shown in his feed. I know there is a feature where you can tell youtube not to show videos from a particular channel but even when doing that for a while, other channels come up with languages I do not want him to learn. Is there any patch that does what I am asking for? 

Thank you!

**Comment by oSumAtrIX** [comment, marked as answer]
No there is no such patch and would need server side filtering
